docs/user-guide/gitops.md

```diff
@@ -34,9 +34,9 @@ GitOps repository is added in two steps:
 
 Once the GitOps repository is added to the platform, you can set custom parameters for the deployed Helm Chart. To redefine the parameters, follow the steps below:
 
-1. Within the GitOps repository, follow the naming convention `<pipeline-name>/<stage-name>/<application-name>-values.yaml` to create the chart values file.
+1. Within the GitOps repository, follow the naming convention `<pipeline-name>/<stage-name>/<application-name>-values.yaml` to create the chart values file. For example, in this case, the file `mypipe/dev/fastapi-demo-values.yaml` will be created.
```

Comment:
, e.g. `mypipe/dev/fastapi-demo-values.yaml`.

Reply:
we can remove

For example, in this case, the file `mypipe/dev/fastapi-demo-values.yaml` will be created.

and just put: `, e.g. mypipe/dev/fastapi-demo-values.yaml`